apps/s_time.c

```diff
@@ -383,8 +347,10 @@ static SSL *doConnection(SSL *scon, const char *host, SSL_CTX *ctx)
 {
     BIO *conn;
     SSL *serverCon;
-    int width, i;
-    fd_set readfds;
+    int i;
+#if defined(SOL_SOCKET) && defined(SO_LINGER)
+    struct linger no_linger;
+#endif
```

Comment:
Why not make portion that makes the call to a block and declare no_linger there?